examples/samv71_xplained_ultra/dac/main.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023, Christopher Durand
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <modm/board.hpp>
+
+using namespace Board;
+
+/**
+ * Simple DAC example
+ *
+ * Output sawtooth waveform in free-running mode.
+ *
+ * See DMA examples for use of DAC with DMA.
+ */
+
+using Out0 = GpioB13;
+
+void waitForTxReady()
+{
+	Dac::Interrupt_t flags{};
+	do
+	{
+		flags = Dac::readInterruptFlags();
+	}
+	while (!(flags & Dac::Interrupt::TxReadyCh0));
```

Comment:
:+1:, I have changed the example from using both channels to only channel 0 because the pin was not usable on the eval board. The code is a leftover from that.